src/components/Book.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const mapDispatchToProps = dispatch => ({
+  removeBook: ID => {
+    dispatch({
+      ID,
+      type: 'REMOVE_BOOK',
+    });
+  },
+});
+
```

Comment:
- [ ] Kindly avoid dispatching actions in the Book component. Please move the dispatch action to the BookForm container. It's bad practice for a component to receive props and also dispatch actions.

Reply:
Done.